prime-router/build.gradle.kts

```diff
@@ -4,13 +4,12 @@ Build script for Prime Router.
 Properties that can be overridden using the Gradle -P arguments:
   DB_USER - Postgres database username (defaults to prime)
   DB_PASSWORD - Postgres database password (defaults to changeIT!)
-  DB_URL - Postgres database URL (defaults to jdbc:postgresql://localhost:5432/prime_data_hub)
+  DB_URL - Postgres database URL (defaults to jdbc:postgresql://172.17.0.1:5432/prime_data_hub, which is the gateway address for the docker 'bridge' network, i.e. your local machine)
```

Comment:
This breaks running the code locally.  We should talk about how these changes are going to change development.

Reply:
This should not break anything unless you do not have docker (which _is_ part of our toolchain anyway). Even if you run PostgreSQL not in docker (i.e. straight on your machine), this will still work (I just validated this with `nc`). To the best of my understanding, for all intents and purposes, `172.17.0.1` ends you up at the same place ends you at `127.0.0.1`.